sklego/meta/regression_outlier_detector.py

```diff
@@ -112,7 +129,8 @@ def fit(self, X, y=None):
         ValueError
             If the `model` is not a regression estimator.
         """
-        self.idx_ = np.argmax([i == self.column for i in X.columns]) if isinstance(X, pd.DataFrame) else self.column
+        X = nw.from_native(X, strict=False)
```

Comment:
the question of how expensive this is came up on the livestream

on my laptop, it's about 1-2 microseconds

Reply:
Can we pitch that as it is significantly less than converting to pandas 😁?

Reply:
oh yeah! even for just a small <1000 row dataframe, it takes 700-1200 times longer (depending on whether you use pyarrow)

and that's without counting that converting to pandas would break polars' optimisations, use much more memory, "force" users to have pandas+pyarrow as dependencies...I think we can sell this well :)